index.js

```diff
@@ -14,6 +15,16 @@ function store () {
       })
     }
 
+    emitter.on(state.events.RENDER, function () {
+      onIdle(function () {
```

Comment:
We can probably allocate this closure statically outside the emitter callback, saving some GC work (:

Reply:
Lol, the irony! But your absolutely right, I'll look into it.